NEW_README.md

```diff
@@ -4,6 +4,30 @@
 [![Godoc](https://img.shields.io/badge/go-docs-blue?logo=go&logoColor=white&label=%20&labelColor=gray&color=blue)](https://amonks.github.io/run)
 [![Go Report Card](https://goreportcard.com/badge/github.com/amonks/run)](https://goreportcard.com/report/github.com/amonks/run)
 
-Run is a task runner that simplifies executing and managing tasks defined in `tasks.toml` files, with a focus on development workflows. It features an interactive UI for long-lived processes and a non-interactive output mode for short-lived tasks, making it versatile for a wide range of development tasks.
+RUN is a task runner that simplifies executing and managing tasks defined in `tasks.toml` files. It provides a versatile set of features making it well suited for a range of use cases, from simple build scripts to complex development workflows.
 
 <img alt="interactive TUI" src="screenshots/tui.gif?raw=true" />
+
+## Features
+
+- **Flexible Task Configuration**: Support for complex task dependencies, environment variable injection, and file watching for automatic task re-execution.
+- **Interactive TUI**: Full mouse support for managing long-lived tasks.
+- **Non-Interactive Output**: Interleaved output suitable for short-lived processes.
+- **Go API**: Extensibility through a Go programming interface.
+
+## Installation
+
+RUN can be installed as a single binary or via the Go command line tool:
+
+### **Pre-compiled binary:**
+
+Download the latest release from the [releases page](https://github.com/amonks/run/releases), and extract the binary to a directory in your `PATH`. This can be done in a single command like:
+
+    $ curl -sL https://github.com/amonks/run/releases/download/<RELEASE_VERSION>/run_<RELEASE_ARCH>.tar.gz | tar -x && chmod +x run && mv run ~/go/bin
```

Comment:
I like the idea of a one-liner for installation, but in this implementation, readers have to manually edit it to suit their needs (eg is ~/go/bin on their path? version numbers?) 

I think understanding and editing the script is maybe harder than using the releases page.

I added #94 to track the idea of a one-liner install process.